### PR TITLE
S1 d

### PR DESCRIPTION
**Pull request by AlisonYoon**

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -0,0 +1,34 @@
+<?php
+
+$db = new PDO('mysql:host=192.168.20.20; dbname=collectorsapp', 'root', '');
+$db->setAttribute(
+    PDO::ATTR_DEFAULT_FETCH_MODE,
+    PDO::FETCH_ASSOC);
+
+$sql = 'SELECT * FROM `grocery_item`';
```
Comment (mallowmew) [inline index.php]: You dont need this if you're using prepare()

File: index.php [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+$sql = $db->prepare('SELECT * FROM `grocery_item`;');
```
Comment (mallowmew) [inline index.php]: I would suggest putting your query -> execute -> output process into a function, rather than inline.

File: index.php [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+$sql->execute();
+$groceryItem = $sql->fetchAll();
+
+//var_dump($groceryItem);
+
+foreach($groceryItem as $item) {
```
Comment (mallowmew) [inline index.php]: Similarly, I would suggest putting your display as html into a function as well, so it can be unit tested.

File: index.php [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+$db->setAttribute(
+    PDO::ATTR_DEFAULT_FETCH_MODE,
+    PDO::FETCH_ASSOC);
+
+$sql = 'SELECT * FROM `grocery_item`';
+$sql = $db->prepare('SELECT * FROM `grocery_item`;');
+$sql->execute();
+$groceryItem = $sql->fetchAll();
+
+//var_dump($groceryItem);
```
Comment (mallowmew) [inline index.php]: Remove commented lines

Comment (AlisonYoon) [reply index.php]: ok

File: index.php (hunk 3 above)
Comment (AlisonYoon) [reply index.php]: ok changed to function

File: index.php (hunk 1 above)
Comment (AlisonYoon) [reply index.php]: ok

File: index.php (hunk 2 above)
Comment (AlisonYoon) [reply index.php]: ok did it thanks

File: function.php [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+    }
+    return $itemRow;
+}
+
+
+//function call
+
+$db = connectDB();
+$groceryItems = getData($db);
+$groceryItemsArray = processData($groceryItems);
```
Comment (mallowmew) [inline function.php]: Do these go in index.php?

Comment (AlisonYoon) [reply function.php]: yes changed it

**Pull request merged**